Assets/MixedRealityToolkit/Utilities/MixedRealityToolkitSceneChecker.cs

```diff
@@ -0,0 +1,130 @@
+                    break;
+            }
+        }
+
+        private static void CheckMixedRealityToolkitScene()
+        {
+            // Create The MR Manager if none exists.
+            if (!MixedRealityToolkit.IsInitialized)
+            {
+                // Search the scene for one, in case we've just hot reloaded the assembly.
+                var managerSearch = Object.FindObjectsOfType<MixedRealityToolkit>();
+
+                if (managerSearch.Length == 0)
+                {
+                    HideNoActiveToolkitWarning = SessionState.GetBool(HideNoActiveToolkitWarningKey, false);
+                    if (!HideNoActiveToolkitWarning)
+                    {
+                        NoActiveToolkitWarning.OpenWindow(null);
+                    }
+                    return; 
+                }
+            }
+        }
+
+        internal class NoActiveToolkitWarning : EditorWindow
+        {
+            private static NoActiveToolkitWarning activeWindow;
+            [SerializeField]
+            public MixedRealityToolkitConfigurationProfile configurationProfile;
+            private bool hideWarning = false;
+
+            public static void OpenWindow(MixedRealityToolkitConfigurationProfile configurationProfile)
+            {
+                // If we already have an active window, bail
+                if (activeWindow != null)
+                {
+                    return;
+                }
+
+                activeWindow = EditorWindow.GetWindow<NoActiveToolkitWarning>();
+                activeWindow.configurationProfile = configurationProfile;
+                activeWindow.maxSize = new Vector2(400, 140);
+                activeWindow.minSize = new Vector2(400, 140);
+                activeWindow.titleContent = new GUIContent("No Active Toolkit Found");
+
+                activeWindow.ShowPopup(); 
+            }
+
+            private void OnGUI()
+            {
+                EditorGUILayout.HelpBox("There is no active Mixed Reality Toolkit in your scene. Would you like to create one now?", MessageType.Warning);
+
+                hideWarning = EditorGUILayout.Toggle("Don't show this again", hideWarning);
+
+                configurationProfile = (MixedRealityToolkitConfigurationProfile)EditorGUILayout.ObjectField(configurationProfile, typeof(MixedRealityToolkitConfigurationProfile), false);
+                if (configurationProfile == null)
+                {
+                    EditorGUILayout.HelpBox("Select a configuration profile.", MessageType.Info);
+                }
+
+                EditorGUILayout.BeginHorizontal();
+                GUI.enabled = (configurationProfile != null);
+                if (GUILayout.Button("Yes"))
+                {
+                    var playspace = MixedRealityToolkit.Instance.MixedRealityPlayspace;
+                    Debug.Assert(playspace != null);
+                    MixedRealityToolkit.Instance.ActiveProfile = configurationProfile;
```

Comment:
@davidkline-ms Do you think this use of `MixedRealityToolkit.Instance` needs to be changed to conform to your own branch?

Reply:
@lukastoenneMS, i'll handle the updates as I merge into my branch/fork. that way, this PR stays as targeted as possible

Reply:
I'll handle the update when i merge into my working branch. thanks!